lib/tasks/rummager.rake

```diff
@@ -1,5 +1,26 @@
 namespace :rummager do
   task :index => "panopticon:register" do
-    # Placeholder until the deploy scripts have been updated
+    Rake::Task["rummager:index_special_routes"].invoke
+  end
+
+  task :index_special_routes => :environment do
+    require 'gds_api/rummager'
+    rummager = GdsApi::Rummager.new(ENV["RUMMAGER_HOST"] || Plek.current.find('search'))
+
+    rummager_payloads = [
+      {
+        title: "Find your local council",
+        description: "Find your local authority in England, Wales, Scotland and Northern Ireland",
+        link: "/find-local-council",
+        indexable_content: "Find your local authority in England, Wales, Scotland and Northern Ireland",
+        public_timestamp: Time.zone.now.iso8601,
+      },
+    ]
+
+    rummager_payloads.each do |rummager_payload|
+      # custom-application is ultimately what the /help route ends up as when it
```

Comment:
How do 'custom-application' and the '/help' route relate to this bit of code? Maybe the comment could add some more context?